RecoLocalTracker/SiStripClusterizer/plugins/ClustersFromRawProducer.cc

```diff
@@ -424,7 +424,7 @@ void ClusterFiller::fill(StripClusterizerAlgorithm::output_t::TSFastFiller& reco
             //rawAlgos_->suppressor->suppress( digis, zsdigis);
             uint16_t firstAPV = ipair * 2;
             rawAlgos.suppressVirginRawData(id, firstAPV, digis, zsdigis);
-            for (const auto digi : zsdigis) {
+            for (const auto& digi : zsdigis) {
```

Comment:
this is coupled to the change made in #29568 and will not be necessary after that PR is merged.
please revert

Reply:
thanks @slava77 , I will merge the two PRs